### DISCUSSION TITLE
Cook mode button

### DISCUSSION THREAD

**Discussion by Toolhead** [mealie-recipes/mealie#4296] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

Computer goes to sleep.

### Please provide a concise description of the feature that would resolve your issue.

Several recipe websites have a button "cook mode" that prevents the screen to go dark.
I would like to see that in Mealie.

### Please consider and list out some caveats or tradeoffs made in your design decision

none

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by Kuchenpirat** [comment]
If supported there is a "Keep Screen Awake" Toggle at the bottom of the recipe pages and the shopping list. 

To be supported you must access your instance via https.

**Reply by Toolhead** [reply]
Ah, thanks.
It can be closed now.
